test/hare_test

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+import logging
+
+from enum import Enum
+
+class Plan(Enum):
+    Sanity = 'sanity'
+    Regression = 'regression'
+    Full = 'full'
+    Performance = 'performance'
+    Scalability = 'scalability'
+
+def setup_logging():
+    logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+def add_plan_argument(parser):
+    parser.add_argument('--plan',
+                        help='Testing plan to be executed. Supported '
+                            'values:' + str([e.value for e in Plan]),
+                        required=True,
+                        nargs=1,
+                        type=Plan,
+                        action='store')
+    return parser
+
+def execute(args):
+    logging.info('Executing plan (%s)', args.plan[0].value)
+
+def main():
+    p = argparse.ArgumentParser(description='Hare test suite')
```

Comment:
All this could have happened at the level of `hare_setup test`. I am not sure that this script is actually needed.

Reply:
We do have some vaidation like plan type etc but actul work is done here, so I think we should have arguments and supported values defined here as this is standalone rpm.

Reply:
@SwapnilGaonkar7 I'm afraid that "actual work" is nothing more than invoking a command like this 
`pytest <path to folders with tests> -E production -m <plan>`

And that command could have been issued right from `hare_setup` script. The only thing that `test` rpm brings is a set of test scripts that are deployed to `<path to folders with tests>`

Let's keep simple things simple.

Reply:
Agree, let's not have multiple command line interfaces. We can just have a test package with set of tests and `hare_setup test` would invoke them. We just need to make sure that a proper error message is generated in case the test cases are not available at the given path instead of ignoring or failing silently.